migrations/versions/fe2cf1f4845e_.py

```diff
@@ -0,0 +1,55 @@
+"""empty message
```

Comment:
chyba bedzie trzeba dac do .gitignore zawartość /versions? albo nie commitować tego

Reply:
Versions musi być bo są tam migracje bo bazy danych. Używane przy make `database.*`

Reply:
wiem że musza tylko pytanie czy musimy je mieć na gicie? bo sciagnalem sobie brancha i nie moglem i tak migracji zrobic (ani upgrade ani migrate) bo wersje miales w swojej bazie pewnie :)

Reply:
Moim zdaniem powinno to być. Kwestia tylko w jaki sposób to mergować i używać

Reply:
https://stackoverflow.com/questions/55715129/work-on-multiple-branches-with-flask-migrate

Reply:
powinny być pliki migracji, nie może być to w gitignore, bez tego nie utrzymamy aktualizacji bazy